cmake/Dependencies.cmake

```diff
@@ -1896,6 +1896,8 @@ if(USE_KINETO)
       set(CUPTI_LIB_NAME "cupti.lib")
     endif()
 
+    set(NVPERF_HOST_LIB_NAME "libnvperf_host.so")
```

Comment:
can you follow the libcupti pattern above so that this works with Windows too? And I'm not sure if this has implications for binary packaging (in which case we need to link statically; cc @malfet)